### PR TITLE
Lint and pretty fixes + tests use next validator

### PR DESCRIPTION
**Pull request by joewagner**
@carsonfarmer I got the tests passing with a build of the validator that includes the testnet/mainnet split.  I also fixed the lint and prettier errors

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -0,0 +1 @@
+coverage/*
```
Comment (carsonfarmer) [inline .eslintignore:1]: Rather that having another admin file, I was opting for "ignorePatterns" in the eslintrc.json file. I'm going to merge this in, and then revert to using that if you don't mind?

File: src/validator/client/README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 # 📘️ openapi-typescript-fetch
 
-A typed fetch client for [openapi-typescript](https://github.com/drwpow/openapi-typescript) 
+A typed fetch client for [openapi-typescript](https://github.com/drwpow/openapi-typescript)
```
Comment (carsonfarmer) [inline src/validator/client/README.md:5]: This whole package is actually "vendor" from a separate project, so I had opted to _not_ lint or format it. Actually, it should have been added to the ignored patterns for prettier and eslint. Buuuut, it looks like all the issues are auto-fixable, so I'll allow it in this case 😉 . Good idea.

**Review by carsonfarmer — APPROVED** [review]

**Comment by carsonfarmer** [comment]
I hope you don't mind @joewagner, but I'm going to merge this PR into my branch. Thanks for the fixes!

**Comment by joewagner** [comment]
> I hope you don't mind @joewagner, but I'm going to merge this PR into my branch. Thanks for the fixes!

@carsonfarmer perfecto.
Nice work on the refactor it's looking good!!

**Pull request merged**